clients/java/src/main/java/io/camunda/client/api/command/CompleteJobResult.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.client.api.command;
+
+import java.util.List;
+
+public class CompleteJobResult {
+
+  private String assignee;
+  private String dueDate;
+  private String followUpDate;
+  private List<String> candidateGroups;
+  private List<String> candidateUsers;
+  private int priority;
+
+  public CompleteJobResult() {}
```

Comment:
❓ @ana-vinogradova-camunda @korthout
What do you think about adding a `deny(boolean isDenied)` method to this class as well, to support the full range of job result functionality?

Reply:
Great point! We missed it, and added it now